service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/MhsRequestBuilder.java

```diff
@@ -70,4 +71,25 @@ private WebClient buildWebClient(HttpClient httpClient) {
             .defaultUriVariables(Collections.singletonMap("url", mhsConfiguration.getUrl()))
             .build();
     }
+
+    public RequestHeadersSpec<?> buildSendAcknowledgement(String requestBody, String fromOdsCode, String conversationId) {
+        SslContext sslContext = requestBuilderService.buildSSLContext();
+        HttpClient httpClient = HttpClient.create().secure(t -> t.sslContext(sslContext));
+        WebClient client = buildWebClient(httpClient);
+
+        WebClient.RequestBodySpec uri = client
+            .method(HttpMethod.POST)
+            .uri(mhsConfiguration.getUrl());
+
+        BodyInserter<Object, ReactiveHttpOutputMessage> bodyInserter = BodyInserters.fromValue(requestBody);
+
+        return uri
+            .accept(APPLICATION_JSON)
+            .header(ODS_CODE, fromOdsCode)
+            .header(CONTENT_TYPE, APPLICATION_JSON_VALUE)
+            .header(INTERACTION_ID, MHS_OUTBOUND_ACKNOWLEDGEMENT_INTERACTION_ID)
+            .header(WAIT_FOR_RESPONSE, FALSE)
+            .header(CORRELATION_ID, conversationId)
```

Comment:
Need to specify a uuid (generate a new/random one) since MHS will not tell us the id it otherwise generates
```suggestion
            .header(CORRELATION_ID, conversationId)
            .header(MESSAGE_ID, randomUuid)
```